package/cached_repository/lib/resource.dart

```diff
@@ -49,18 +49,23 @@ class Resource<T> {
           state == other.state &&
           _compareData(data, other.data) &&
           message == other.message &&
-          error == other.error;
+          error == other.error &&
+          stackTrace == other.stackTrace;
 
   bool _compareData(dynamic v1, dynamic v2) =>
-      v1 is List && v2 is List ? listEquals(v1, v2) : v1 == v2;
+      v1 is List && v2 is List ? const ListEquality().equals(v1, v2) : v1 == v2;
```

Comment:
what is the difference?

Reply:
Both snippets are equivalent and archive the same result. In the project the first one was with false)) for package))